api/shared-modules/src/node/utils/encryption/index.ts

```diff
@@ -48,3 +50,19 @@ export const decrypt = (cipher: string, secret: string) => {
 	const decrpyted = Buffer.concat([decipher.update(Buffer.from(hash, 'hex')), decipher.final()]);
 	return decrpyted.toString();
 };
+
+export const asymEncrypt = (data: any, privateKey: string, channelPublicKey: string): string => {
+	const sharedKey = X25519.keyExchange(bs58.decode(privateKey), bs58.decode(channelPublicKey));
```

Comment:
could you please make a short research if we can find a node library supporting this so we don't have the dependency to the identity kit